client_taxi_driver/shared/src/commonMain/kotlin/domain/ErrorType.kt

```diff
@@ -17,4 +17,12 @@ class ServerSideException(message: String) : BpException(message)
 
 class NotFoundedException : BpException("Not founded")
 
-class UnKnownErrorException(message: String) : BpException(message)
\ No newline at end of file
+class UnKnownErrorException(message: String) : BpException(message)
```

Comment:
```suggestion
class UnknownErrorException(message: String) : BpException(message)
```
unknown is written as a single word not two workds